demo/src/screens/componentScreens/CarouselScreen.js

```diff
@@ -73,7 +73,7 @@ class CarouselScreen extends Component {
         </View>
 
         <View padding-20>
-          <Carousel migrate containerStyle={{height: 160}} initialPage={INITIAL_PAGE} loop>
+          <Carousel migrate containerStyle={{height: 160}} initialPage={INITIAL_PAGE} loop usePagination>
```

Comment:
what's `usePagination`? I don't you defined this prop anywhere..

Reply:
opps. Fixed